components/scoreHandler.ts

```diff
@@ -679,3 +679,16 @@ export async function missionEnd(
         data: result,
     })
 }
+
+/**
+ * For testing purposes only
+ */
+interface Testing {
+    calculatePlaystyle: typeof calculatePlaystyle
+}
+
+export const _testing: Testing = <Testing>{}
+
+if (PEACOCK_TEST) {
+    _testing.calculatePlaystyle = calculatePlaystyle
+}
```

Comment:
This feels like a bad idea.

Reply:
Yep, but if you don't want people to complain about removing @internal methods they did end up using, this is the way. They will simply never get a reference when Peacock is run normally. This is probably best compared to the internal keyword found in, for example, C#. Where everything within the code-base is allowed to access it.